lib/code_corps_web/views/task_view.ex

```diff
@@ -37,3 +37,27 @@ defmodule CodeCorpsWeb.TaskView do
     status
   end
 end
+
+defmodule CodeCorpsWeb.TaskIncludedView do
+  @moduledoc false
+  use CodeCorpsWeb, :view
+  use JaSerializer.PhoenixView
+
+  def type, do: "task"
+
+  attributes [
+    :archived, :body, :created_at, :created_from, :has_github_pull_request, :inserted_at, 
+    :markdown, :modified_at, :modified_from, :number, :order, :status, :title, :updated_at
+  ]
+
+  has_one :github_issue, type: "github-issue", field: :github_issue_id, include: true
+  has_one :github_pull_request, serializer: CodeCorpsWeb.GithubPullRequestView, include: true
+  has_one :github_repo, type: "github-repo", field: :github_repo_id
+  has_one :user, serializer: CodeCorpsWeb.UserIncludedView, include: true
+  has_one :user_task, serializer: CodeCorpsWeb.UserTaskView, include: true
+
```

Comment:
This task now includes all relevant relationships I believe that are needed for the task index route.